.env.example

```diff
@@ -1,6 +1,8 @@
 NEW_EXPENSIFY_URL=https://new.expensify.com/
 SECURE_EXPENSIFY_URL=https://secure.expensify.com.dev/
 EXPENSIFY_URL=https://www.expensify.com.dev/
+STAGING_EXPENSIFY_URL=https://staging.expensify.com.dev/
+STAGING_SECURE_EXPENSIFY_URL=https://staging-secure.expensify.com.dev/
```

Comment:
I am unsure about these, can you give a bit more context, we dont have any `staging` on local `dev` machine

Reply:
I think the whole PR is to address an issue for open source contributors and enable them to switch and use the staging server to test stuff like bank accounts (otherwise we use PROD and can't use fake/mock accounts) 

Not sure if we need to add anything to the .env - internal devs don't use the proxy server, and only the proxy server uses these values so far

Reply:
Nope `STAGING_EXPENSIFY_URL` should always point to staging, I guess that's why it's removed from the `.env`

Internal devs only change the DEFAULT API when the use the `.dev` server, they never change the STAGING API and use the same STAGING API as external devs

Reply:
I think it would be best to just drop the change, because it's not needed

Reply:
@mountiny do internal developers have the same problem as outlined in the issue: 
- https://github.com/Expensify/App/issues/12315

Do you need to use the STAGING API as well to overcome this problem?